### DISCUSSION TITLE
Add ability to filter RSS by a combination of username and tag

### DISCUSSION THREAD

**Discussion by berviantoleo** [forem/forem#19219] [Feature Requests]
I can filter RSS by a username. I also can filter RSS by a tag. But, how to combine those filters?

* Main feed: [dev.to/feed/](https://dev.to/feed/)
* Author feed: [dev.to/feed/[username]](https://dev.to/feed/%5Busername%5D)
* Tag feed: [dev.to/feed/tag/[tag]](https://dev.to/feed/tag/%5Btag%5D)

I think it's not allowed by this logic: https://github.com/forem/forem/blob/main/app/controllers/articles_controller.rb#L38.

**Comment by AlexaHills** [comment]
Go to the search bar or advanced search options on the platform. Type in the username you want to filter by in the search bar, and hit enter. Look for an option to add a filter or refine your search. This may be a button or drop-down menu. Select the option to filter by tag or category, and enter the tag or category you want to filter by. Click on the search button or hit enter to search for items that match both your username and tag filters.

**Reply by berviantoleo** [reply]
Thank you for answering my question.

But, sorry, I didn't ask clearly. I mean for the RSS feeds.

**Comment by AlexaHills** [comment]
To combine filters in an RSS feed by both a username and a tag, you can use the following syntax
https://www.example.com/rss?user=username&tag=tagname Replace username with the desired username and tagname with the desired tag. This will filter the RSS feed to include only items that match both the specified username and tag. If the RSS feed URL already includes a username or tag filter, you can add the additional filter by using the & character to separate the filters. 
https://www.example.com/rss?user=username&tag=tagname&category=categoryname You can also check here https://decortrendy.com/
